src/pal/inc/pal.h

```diff
@@ -4474,6 +4474,7 @@ PALIMPORT LONG __cdecl labs(LONG);
 // clang complains if this is declared with __int64
 PALIMPORT long long __cdecl llabs(long long);
```

Comment:
Can you please move the llabs above the `#ifndef PAL_STDCPP_COMPAT`? That way we would have just one #ifndef there.

Reply:
Done